### PR TITLE
Development

### PR DESCRIPTION
**Pull request by farouqdaif1**
* Chose an API for the project
* Followed design guidelines
* Interactions:
* Home page: When the page loads, the web app shows the Country's name and recovered cases and When * * the user clicks (or taps) on a category item, the application navigates to the details page.
* Details page: the web app retrieves data from the API to show detailed data in the category and When * the user clicks on the "Back" button (<), the user navigates to the home page.
* Created unit and integration tests.
* The project is a single-page application (SPA) built with React and Redux.
* Every page (the main page and the pages for each item) have a unique route within the SPA.
* The project is deployed and accessible online (added a link to the online version of your app to * README file).
* Recorded a video for documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,84 @@
...
+
+## Screenshots
+
+- Home Page
+![screenshot](./app_screenshot.png)
+- Details Page
+![screenshot](./app_screenshot1.png)
+
+
+## Video
```
Comment (SajjadAhmad14) [inline README.md:15]: Please record a new video  and highlight about a piece of the code that you find interesting or is proud of making. You do not need to present the whole project again but just talk about the piece of code which you are proud of.

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (SajjadAhmad14) [inline public/manifest.json:1]: Please remove all the boilerplate code which is created by `Create React App`from `public` directory.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Farouq, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by farouqdaif1** [comment]
thanks

File: public/manifest.json (hunk 2 above)
Comment (farouqdaif1) [reply public/manifest.json:1]: thanks

File: README.md (hunk 1 above)
Comment (farouqdaif1) [reply README.md:15]: thanks

**Review by rukundoeric — APPROVED** [review]
Hi @farouqdaif1,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**